tests/conformance/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "require": {
+      "google/cloud-functions-framework": "*"
```

Comment:
Need to change:

"google/cloud-functions-framework": "https://github.com/GoogleCloudPlatform/functions-framework-php/#commit=abc"

Need to pass SHA to this file.